include/syntax/php

```diff
@@ -0,0 +1,381 @@
+// supported feature lists -----------------------------------------------------
+
+supported_apis = ["simple", "generic", "record"];
+supported_api_styles = ["free-form"];
+supported_code_models = ["loop-switch"];
+supported_targets = ["code", "dot"];
+supported_features = ["nested-ifs", "bitmaps", "tags", "captures", "captvars"];
```

Comment:
For all supported and unsupported features there must be a test. Tags, captures and captvars are covered by examples, and for the rest see tests in `test/codegen/<language>`.